src/vma/proto/flow_tuple.cpp

```diff
@@ -161,7 +161,7 @@ bool flow_tuple::is_3_tuple()
 void flow_tuple::set_str()
 {
 	/* cppcheck-suppress wrongPrintfScanfArgNum */
-	snprintf(m_str, STR_MAX_LENGTH, "dst:%d.%d.%d.%d:%d, src:%d.%d.%d.%d:%d, protocol:%s",
```

Comment:
you can also use
"dst:%hd.%hd.%hd.%hd:%d, src:%hd.%hd.%hd.%hd:%d, protocol:%s"
and then leave the size of STR_MAX_LENGTH at 100